server/src/database/queries/addUdemyCourse.js

```diff
@@ -0,0 +1,31 @@
+const connection = require('../config/connection');
+
+const addUdemyCourse = (data) => {
+  const {
+    title,
+    categoryId,
+    image_480x270: image,
+    author,
+    url,
+    rating,
+    num_reviews: reviews,
+    objectives_summary: description,
+    source,
+  } = data;
+  return connection.query(
+    'INSERT INTO course (category_id, title, image, author_name, url, rate, reviews, description, source) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)',
+    [
+      categoryId,
+      title,
+      image,
+      author,
+      url,
+      rating,
+      reviews,
+      description[0],
```

Comment:
here, try to concatenate between all details to achieve a good description : 
I mean all elements here
```
0: "Understand the real reasons why they buy from you"
1: "Know what to say in your marketing"
2: "Create ads and copy that drive sales" 
```